swagger/v1/court_case.json

```diff
@@ -0,0 +1,76 @@
+{
+  "CourtCase": {
+    "type": "object",
+    "required": [
+      "id",
+      "type",
+      "attributes"
+    ],
+    "properties": {
+      "id": {
+        "type": "string",
+        "format": "uuid",
+        "example": "3fa9c528-3301-4d69-f200-69e482ce1ed8",
+        "description": "The unique identifier (UUID) of this object"
+      },
+      "type": {
+        "type": "string",
+        "example": "court_cases",
+        "enum": ["court_cases"],
+        "description": "The type of this object - always `court_cases`"
+      },
+      "attributes": {
+        "type": "object",
+        "required": [
+          "nomis_case_id",
+          "nomis_case_status",
+          "nomis_case_start_date",
+          "nomis_case_type",
+          "nomis_case_number"
+        ],
+        "properties": {
+          "nomis_case_id": {
+            "type": "string",
+            "example": "T20167984",
```

Comment:
the example is actually a case_number, it should be a case_id.

Reply:
👍